packages/components/src/hooks/useCommonTranslation.ts

```diff
@@ -1,6 +1,8 @@
 import { useTranslation } from 'next-i18next';
 import { commonConfig } from '../translations/common.config';
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
is this really needed?

Reply:
hmmm.. at some point it was, but seems like not anymore. Good catch.